src/components/search-form/search-form.test.js

```diff
@@ -0,0 +1,79 @@
+import { MemoryRouter } from 'react-router-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SearchForm } from './search-form';
+import { useGetParams } from '../../hooks/use-get-params/use-get-params';
+
+jest.mock('../../hooks/use-get-params/use-get-params', () => ({
+  useGetParams: jest.fn(() => [{}]),
+}));
+
+describe('Search input', () => {
+  it('Matches the snapshot', () => {
+    render(
+      <MemoryRouter>
+        <SearchForm />
+      </MemoryRouter>
+    );
+
+    const searchInputElement = screen.getByRole('searchbox');
+
+    expect(searchInputElement).toMatchSnapshot();
+  });
+
+  it('Can contain a value', async () => {
```

Comment:
Not just can, it must contain value.

Reply:
Changed the description to `'Must contain the entered value'`.